Apps/W1/EDocumentConnectors/Tietoevry/app/src/Upgrade.Codeunit.al

```diff
@@ -0,0 +1,64 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Tietoevry;
+
+using System.Upgrade;
+#if not CLEAN26
+using Microsoft.eServices.EDocument.Integration;
+using Microsoft.eServices.EDocument;
+#endif
+
+codeunit 6390 Upgrade
```

Comment:
You dont need this. This app will only work on version 2. :)

Reply:
Removed Upgrade Codeunit.